loadtest/control/simulcontroller/controller.go

```diff
@@ -58,8 +59,10 @@ func (c *SimulController) Run() {
 		return
 	}
 
+	semCount := runtime.GOMAXPROCS(0)
```

Comment:
I am not sure `GOMAXPROCS` is what we need here. Potentially we could have hundreds of users typing so I would make this reasonably high.

Reply:
I picked that one because it gives a hint about the system capability. Anyway, I'm open to suggestions 🤗

Reply:
That's reasonable but we should probably add a multiplier.